README.md

```diff
@@ -26,11 +26,11 @@ allprojects {
 }
 ```
 
-And this lines to your main project
+And these lines to your main project
 ```
 dependencies {
-  implementation 'com.conviva.sdk:conviva-core-sdk:2.145.1' // <-- conviva sdk
-  implementation 'com.bitmovin.analytics:conviva:1.1.1'
+  implementation 'com.conviva.sdk:conviva-core-sdk:4.0.20' // <-- conviva sdk
+  implementation 'com.bitmovin.analytics:conviva:2.0.2'
```

Comment:
Update this file to replace `ConvivaAnalytics` with `ConvivaAnalyticsIntegration`

Reply:
This is done in [this commit](https://github.com/bitmovin/bitmovin-player-android-analytics-conviva/pull/38/commits/80618197309f0e3441dd5254221fcfac33f25d84)